src/AppInstallerCLICore/ChannelStreams.cpp

```diff
@@ -39,6 +39,15 @@ namespace AppInstaller::CLI::Execution
         return *this;
     }
 
+    void BaseStream::Close() 
+    {
+        m_enabled = false;

```

Comment:
If this were a single shared object that every `OutputStream` had a reference to, at least `m_enabled` would need to be updated to be `std::atomic_bool` to prevent a race at `Close`.